docs/src/lib/operations.md

```diff
@@ -449,3 +449,9 @@ to_union_of_projections(::Rectification{N}, ::Bool=false) where {N<:Real}
 ```@docs
 RectificationCache
 ```
+
+## Interior
+
+```@docs
+is_interior_point(::AbstractVector, ::LazySet; p=Inf, ε=_TOL_F64.rtol)
```

Comment:
To fix the [failed docs build](https://travis-ci.org/JuliaReach/LazySets.jl/jobs/584984815), try without specifying the optional keyword arguments, that is:

```julia
is_interior_point(::AbstractVector, ::LazySet)
```

Reply:
No, the problem is that by adding the type parameter, the signature has changed (see [my comment here](https://github.com/JuliaReach/LazySets.jl/pull/1690#discussion_r324427969)).

Reply:
Either way is good. AFAIK optional arguments are *not* mandatory in the `@docs`. If this has changed we have to review [this section](https://juliareach.github.io/JuliaReachDevDocs/latest/guidelines/#Extending-the-online-documentation-1).